backstack/build.gradle.kts

```diff
@@ -14,6 +14,18 @@ kotlin {
   jvm()
   ios()
   iosSimulatorArm64()
+  js(IR) {
+    moduleName = property("POM_ARTIFACT_ID").toString()
+    compilations.all {
+      kotlinOptions {
+        sourceMap = true
+        moduleKind = "umd"
+        metaInfo = true
```

Comment:
I believe UMD is the default now. Source map is probably also generated by default. Not sure what metaInfo is or why you need it.

Reply:
nice. Copied this from a compose js sample project I saw somewhere but can't find it now. [bcc3fd6](https://github.com/slackhq/circuit/pull/801/commits/bcc3fd6954a4d0f5e1c1ea9b85c19bf048808786)